spec/unit/util/dsc/local_configuration_manager_spec.rb

```diff
@@ -63,7 +63,35 @@
         let(:lcm_standard_error) { nil }
         let(:lcm_cmdlet_success) { true }
 
-        it "should successfully return resource information for normally formatted output when cmdlet the cmdlet succeeds" do
+        it "successfully return resource information for normally formatted output when cmdlet the cmdlet succeeds" do
+          test_configuration_result = lcm.test_configuration("config", {})
+          expect(test_configuration_result.class).to be(Array)
+          expect(test_configuration_result.length).to be > 0
+          expect(Chef::Log).not_to receive(:warn)
+        end
+      end
+
+      context "returns successfully for PowerShell version 5" do
```

Comment:
Nitpicking: It's not clear what it returns successfully for PowerShell version 5. It would be a good idea to use:
```
context "when running on PowerShell version 5" do
```

Reply:
it seems like if you end up with a combined `lcm_command`, you would want a test somewhere that ensures you get `-whatif` on ps4 and `test-dscconfiguration` on ps5.